app/assets/stylesheets/geoblacklight.scss

```diff
@@ -1,3 +1,7 @@
 @import 'bootstrap/scss/bootstrap';
 @import 'blacklight-frontend/app/assets/stylesheets/blacklight/blacklight';
-@import '@geoblacklight/frontend/app/assets/stylesheets/geoblacklight/geoblacklight';
+@import '@geoblacklight/frontend/dist/stylesheets/geoblacklight/geoblacklight';
+
+.palo-alto-base {
+  --bs-dark-rgb: 23, 94, 84;
+}
```

Comment:
can we put this in our own `style.css`? I think this stylesheet is just for GBL stuff

Reply:
@thatbudakguy I moved this to the application.css file in the PR for the footer.